From 26bbb85c150f882c05399e4c574208b8b1242082 Mon Sep 17 00:00:00 2001
From: James Le Cuirot <chewi@gentoo.org>
Date: Tue, 7 Apr 2020 19:32:15 +0100
Subject: [PATCH] build: Fix librt and libm dependencies in Meson

They need to be requested without the lib prefix, otherwise a
generated pkg-config file ends up with absolute paths instead of -l
flags.

Signed-off-by: James Le Cuirot <chewi@gentoo.org>
Acked-by: Frediano Ziglio <fziglio@redhat.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/meson.build
+++ b/meson.build
@@ -103,7 +103,7 @@ foreach dep : ['libjpeg', 'zlib']
 endforeach
 
 if host_machine.system() != 'windows'
-  foreach dep : ['librt', 'libm']
+  foreach dep : ['rt', 'm']
     spice_server_deps += compiler.find_library(dep)
   endforeach
 else
